package.json

```diff
@@ -18,18 +18,26 @@
   },
   "dependencies": {
     "@erc725/erc725.js": "0.16.0",
-    "@lukso/lsp-factory.js": "2.4.0",
-    "@lukso/lsp-smart-contracts": "0.7.0",
+    "@lukso/lsp-factory.js": "file:../tools-lsp-factory",
```

Comment:
This was committed by mistake and will be replaced when https://github.com/lukso-network/tools-lsp-factory/pull/180 update is released.